### ISSUE TITLE
Lambda function to handle signature requests?

### ISSUE THREAD

**Issue by scumola** [FineUploader/fine-uploader#1957]
## Type of issue
- [ ] Bug report
- [X] Feature request

## Uploader type
- [ ] Traditional
- [X] S3
- [ ] Azure

#### Feature request details 
Why isn't there just a lambda function that we can drop into AWS with our API keys that fixes the signature issue and doesn't require us to spin up a PHP server to handle the request signatures?  Lambda would be a great solution to this problem.  I'm surprised that this isn't a thing yet.

**Comment by rnicholus** [comment]
This article should provide you the information you need, or at least get you started: http://blog.stratospark.com/secure-serverless-file-uploads-with-aws-lambda-s3-zappa.html

**Comment by scumola** [comment]
Thanks, I read that and tried it but I don't need zappa or letsencrypt or anything like that and frankly I couldn't get it to work because it requires me to upload my AWS creds to an S3 bucket (a no-no), a whole new domain, ssl cert...  I just need a drop-in lambda function that'll do the signature without all of the extra crap.

**Comment by rnicholus** [comment]
Well, I won't have any time to do that in the forseeable future, but I welcome PRs.

**Comment by menangen** [comment]
Yes, I need too guide for using signed url from AWS Lambda on clients side. Which FU build is better to use? S3 or traditional uploader? In which event callback is better to use requesting to Lambda for signed url? And in which step to patch endpoint in FU instance? Advantages/disadvantages of chunked upload to S3, is it possible use chunked feature with one signed url to upload to S3 signed url endpoint? Thx for help :)

**Comment by HemalR** [comment]
Would this be solved by having the signature be created via an arbitrary function? Then people can simply use whatever code they wish in order to generate that signature - whether that is a lambda function or something else (in my case, I'm trying to use a Meteor method to return the valid signature back to the client).
